src/Config.jl

```diff
@@ -18,8 +18,15 @@ function create!(config::Config, content::String)
 end
 Config(content::String) = create!(Config(), content)
 
-function create!(config::Config, d::Dict)
-    str = sprint(JSON.print, d)
+function create!(config::Config, d::Dict; json = false)
```

Comment:
The `create!` functions are not documented in the README so they are purely internal it seems? But then this kwarg doesn't do anything since the default will always be used?

Reply:
It also changes the default from `JSON` to non JSON. Why is that desireable?

Reply:
I have removed the json option in the latest version. I wonder if the JLL might be built without JSON parsing working - I cannot get it to load the test JSON files from the AMGX repository. There is a sort of related issue in the AMGX repository https://github.com/NVIDIA/AMGX/issues/99 where one of the first checks was to see if `RAPIDJSON_DEFINED` is in the build flags. CMake should have found it when building since it is bundled with the AMGX repo. I am able to set most options using the fallback string interface now at least, so the library is functional.